ServerCore/Areas/Identity/Pages/Account/Manage/Index.cshtml.cs

```diff
@@ -75,6 +75,7 @@ public async Task<IActionResult> OnPostAsync()
             Input.ID = thisPuzzleUser.ID;
             Input.IsGlobalAdmin = thisPuzzleUser.IsGlobalAdmin;
             Input.IdentityUserId = thisPuzzleUser.IdentityUserId;
+            Input.MayBeAdminOrAuthor = thisPuzzleUser.MayBeAdminOrAuthor;
```

Comment:
Is there a reason that T-Shirt size is not present here?

Reply:
Not especially, I'll add it in so that it's consistent across pages.